systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -726,4 +729,23 @@ protected void cleanUpKafkaTopics(String componentsNamespaceName) {
         cmdKubeClient(componentsNamespaceName).deleteAllByResource(KafkaTopic.RESOURCE_KIND);
         KafkaTopicUtils.waitForTopicWithPrefixDeletion(componentsNamespaceName, topicName);
     }
+
+    /**
+     * Sets up the namespaces required for the file-based Strimzi upgrade test.
+     * This method creates and prepares the necessary namespaces if operator is installed from example files
+     */
+    protected void fileBasedStrimziUpgradeTestNamespacesSetup() {
```

Comment:
Wouldn't be better to use just (not explicitly mentioning file??)
```suggestion
    protected void setUpStrimziUpgradeTestNamespaces() {
```